analyzers/tests/SonarAnalyzer.Test/TestCases/AspNet/AvoidUnderPosting.CSharp9.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc;
+
+namespace InitProperty
+{
+    public class ModelUsedInController
```

Comment:
Have you considered POCOs defined by a primary constructor? I think it would be worth
* Testing if `class` primary constructors work in MVC binding
* Document `class` primary constructor behavior (probably FNs if supported by the binding mechanism, or FNs otherwise)
* Document `record` primary constructor behavior (probably TPs)